Remotion/Web/Core/Res/Themes/NovaViso/HTML/Common.css

```diff
@@ -132,6 +132,57 @@ input[type="password"].remotion-themed,
   padding-right: var(--remotion-themed-spacing);
 }
 
+.remotion-themed input[type="radio"],
+.remotion-themed input[type="checkbox"]
+{
+  z-index: var(--z-index-hidden);
+  position: absolute;
+  opacity: 0;
+}
+
+.remotion-themed input[type=checkbox] + span
+{
+  display: inline-block;
+}
+
+.remotion-themed input[type=checkbox]:checked + span:before
+{
+  background-image: url('../Image/sprite.svg#CheckBoxTrue');
+}
+
+.remotion-themed input[type=radio]:checked + label:before,
+.remotion-themed input[type=radio]:checked + span:before
+{
+  background-image: url('../Image/sprite.svg#RadioButtonTrue');
+}
+
+.remotion-themed input[type=radio]:disabled + label:before
+{
+  opacity: 0.5;
+  filter: grayscale(100%);
+}
+
+.remotion-themed input[type=checkbox] + span:before
```

Comment:
I believe checkbox and radiobutton would have the same declarations, except the actual image.